s3transfer/download.py

```diff
@@ -192,6 +203,9 @@ def __init__(self, osutil, transfer_coordinator, io_executor,
     def is_compatible(cls, download_target):
         return hasattr(download_target, 'write')
 
+    def buffers_body_in_memory(self):
```

Comment:
Yeah I am good with calling it ``get_task`` or ``get_get_object_task`` tags. I only did it the way I did to make it similar in functionality to the upload input managers which asks if the body gets buffered in memory. But since this is not dependent on if it is a multipart upload or not a multipart upload, I am fine with changing it.